components/Accordion/Accordion.tsx

```diff
@@ -49,8 +63,10 @@ export default function Accordion({
     >
       <div
         role='button'
+        tabIndex={0}
```

Comment:
Does this mean it's always tabIndex='0' for all accordions-panels? Does it matter if we don't set it if they all have the same value?

Reply:
We need to set it for the div to be navigable in the first place. 

Zero just means we are not setting an explicit order, but the items with tabindex zero will be navigated in the order they appear.